cmd/docs.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/speakeasy-api/speakeasy/internal/config"
+	"github.com/speakeasy-api/speakeasy/internal/docsgen"
+	"github.com/speakeasy-api/speakeasy/internal/utils"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs",
+	Short: "Use this command to generate content, compile, and publish SDK docs.",
+	Long:  "Use this command to generate content, compile, and publish SDK docs. This feature is currently in closed beta, please reach out to speakeasy for more information.",
+	RunE:  utils.InteractiveRunFn("Provide a docs sub command"),
+}
+
+var genDocsContentCmd = &cobra.Command{
+	Use:   "content",
```

Comment:
I feel like this should be a verb, especially given that the other commands here will likely be things like `compile` and `publish`. I think `speakeasy docs generate` does the job? A bit similar to `speakeasy generate ...` but I think fine (or that may even be desirable)

Reply:
Hahah I wrestled with the same thing about verbs. Even had `generate` first. That's enough confirmation for me, I'll switch it back